integration/v6/isolated/unset_env_command_test.go

```diff
@@ -0,0 +1,45 @@
+package isolated
+
+import (
+	"fmt"
+
+	"code.cloudfoundry.org/cli/integration/helpers"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gbytes"
+	. "github.com/onsi/gomega/gexec"
+)
+
+var _ = Describe("unset-env command", func() {
+	var (
+		orgName    string
```

Comment:
Looks like you dont need the `var` and the `BeforeEach` blocks as they are not used for the test.